waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -10,8 +10,9 @@ where
 
```

Comment:
I originally rearranged the functions to go with our conventions, but it made the diff too unclear. I'll shuffle them around in a different PR.